File: Java/Conditionals Assignment/Code1.java
import java.util.Scanner;

public class Code1{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter Your Number = ");
        int a = sc.nextInt();

        if(a > 0){
            System.out.println("Number is Positive");
        }
        else if(a == 0){
            System.out.println("Number is 0");
        }
        else{
            System.out.println("Number is Negative");
        }
        sc.close();
    }
}

File: C++/Conditional Statements /Code1.cpp
// Write a C++ program to take a number as input from the user and 
// check whether it is positive, negative, or zero.
// The program should display the result based on the sign of the number
#include<iostream>
using namespace std;

int main(){
    int n;
    cout << "Enter your number = ";
    cin >> n;

    if(n > 0){
        cout << "The number is positive" << endl;
    }
    else if(n == 0){
        cout << "The number is zero" << endl;
    }
    else{
        cout << "The number is negative" << endl;
    }
    return 0;
}